lib/pat.c

```diff
@@ -4357,6 +4376,22 @@ pop(grn_pat_cursor *c)
   return c->sp ? &c->ss[--c->sp] : NULL;
 }
 
+static inline void
+push_pat_cursor(grn_pat_cursor *cursor,
```

Comment:
We should use `pat_cursor` as the prefix for `grn_pat_cursor` methods.

Reply:
I see.
I used `grn_pat_cursir` prefix.